ghost/core/core/server/models/user.js

```diff
@@ -68,7 +68,8 @@ User = ghostBookshelf.Model.extend({
             comment_notifications: true,
             free_member_signup_notification: true,
             paid_subscription_started_notification: true,
-            paid_subscription_canceled_notification: false
+            paid_subscription_canceled_notification: false,
+            mention_notifications: true
```

Comment:
Have you discussed with product about what the default should be?

Reply:
Ref https://ghost.slack.com/archives/C02G9E68C/p1675916876727609